docs/tutorials/tutorial-sql-query-view.md

```diff
@@ -0,0 +1,168 @@
+---
+id: tutorial-sql-query-view
+title: "Tutorial: Use Query view"
+sidebar_label: "Use Query view"
+---
+
+<!--
+  ~ Licensed to the Apache Software Foundation (ASF) under one
+  ~ or more contributor license agreements.  See the NOTICE file
+  ~ distributed with this work for additional information
+  ~ regarding copyright ownership.  The ASF licenses this file
+  ~ to you under the Apache License, Version 2.0 (the
+  ~ "License"); you may not use this file except in compliance
+  ~ with the License.  You may obtain a copy of the License at
+  ~
+  ~   http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing,
+  ~ software distributed under the License is distributed on an
+  ~ "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+  ~ KIND, either express or implied.  See the License for the
+  ~ specific language governing permissions and limitations
+  ~ under the License.
+  -->
+
+
+This tutorial demonstrates some useful features of the Query view in Apache Druid.
+
+You can use the Query view to ingest and query data. You can also use it to test and tune queries before you use them in API requests&mdash;for example, to perform [SQL-based ingestion](../multi-stage-query/api.md).
+
+The tutorial guides you through the steps to ingest sample data, and query the ingested data using some Query view features.
+
+## Prerequisites
+
+Before you follow the steps in this tutorial, download Druid as described in the [quickstart](./index.md) using the [micro-quickstart](../operations/single-server.md#micro-quickstart-4-cpu-16gib-ram) single-machine configuration and have it running on your local machine. You don't need to have loaded any data.
+
+## Run a demo query to ingest data
+
+In this section you load the demo queries that are included in Druid, and run a SQL task to ingest sample data into a datasource.
```

Comment:
CAn we add a little more context here? Something along the lines: Each of the Sample Queries demonstrates a different aspect/feature, for exmple x and y. Each query has detailed comments to help you learn more.

Reply:
Updated.